UI/window-projector.hpp

```diff
@@ -18,7 +18,8 @@ enum class MultiviewLayout : uint8_t {
 	HORIZONTAL_BOTTOM_8_SCENES = 1,
 	VERTICAL_LEFT_8_SCENES = 2,
 	VERTICAL_RIGHT_8_SCENES = 3,
-	HORIZONTAL_TOP_24_SCENES = 4,
+	HORIZONTAL_TOP_18_SCENES = 4,
+	HORIZONTAL_TOP_24_SCENES = 5,
```

Comment:
By changing the constant for 24 scenes in this enum, anyone who has 24 scenes selected will find themselves on the 18 scenes layout when launching OBS after this change.

Unfortunately the settings (incorrectly) uses the saved value as an index on the combox when first loading, despite the items in the combobox using the enum values as data independently of index.

A solution here to both problems would be to keep 24 scenes as 4, add 18 scenes as 5, and then use either findData() or a switch case on the setCurrentIndex in window-basic-settings.cpp.

Reply:
Ah gotcha, made the change back for the order of the entries there.